examples/galaxies/plot_photometry.py

```diff
@@ -0,0 +1,90 @@
+# :meth:`kcorrect.apparent_magnitudes
+# <skypy.galaxy.spectrum.SpectrumTemplates.apparent_magnitudes>`
+# methods to calculate the relevant photometric quantities using the
+# :doc:`speclite <speclite:overview>` package. Note that since the kcorrect
+# templates are defined per unit stellar mass, the total stellar mass of each
+# galaxy must either be given or calculated from its absolute magnitude in
+# another band using :meth:`kcorrect.stellar_mass
+# <skypy.galaxy.spectrum.KCorrectTemplates.stellar_mass>`.
+# An example simulation for the SDSS u- and r-band apparent magnitudes of "red"
+# and "blue" galaxy populations is given by the following config file:
+#
+# .. literalinclude:: ../../../examples/galaxies/sdss_photometry.yml
+#   :language: YAML
+#   :caption: examples/galaxies/sdss_photometry.yml
+#
+# The config file can be downloaded
+# `here <https://github.com/skypyproject/skypy/raw/master/examples/galaxies/sdss_photometry.yml>`_
+# and the simulation can be run either from the command line and saved to FITS
+# files:
+#
+# .. code-block:: bash
+#
+#    $ skypy examples/galaxies/sdss_photometry.yml --format fits
+#
+# or in a python script using the :class:`Pipeline <skypy.pipeline.Pipeline>`
+# class as demonstrated in the `SDSS Photometry`_ section below. For more
+# details on writing config files see the :doc:`Pipeline Documentation </pipeline/index>`.
+#
+# SDSS Photometry
+# ---------------
+#
+# Here we compare the apparent magnitude distributions of our simulated
+# galaxies with data from a :math:`10 \, \mathrm{deg^2}` region of the Sloan
+# Digital Sky Survey [3]_:
+
+from astropy.table import Table, vstack
+from matplotlib import pyplot as plt
+import numpy as np
+from skypy.pipeline import Pipeline
+
+# Execute SkyPy galaxy photometry simulation pipeline
+pipeline = Pipeline.read("sdss_photometry.yml")
+pipeline.execute()
+skypy_galaxies = vstack([pipeline['blue_galaxies'], pipeline['red_galaxies']])
+
+# SDSS magnitude distributions for a 10 degree^2 region
+sdss_data = Table.read("sdss_dered_10deg2.ecsv", format='ascii.ecsv')
+
+# Plot magnitude distributions for SkyPy simulation and SDSS data
+bins = np.linspace(14.95, 25.05, 102)
+plt.hist(skypy_galaxies['mag_r'], bins=bins, alpha=0.5, color='r', label='SkyPy-r')
+plt.hist(skypy_galaxies['mag_u'], bins=bins, alpha=0.5, color='b', label='SkyPy-u')
+plt.plot(sdss_data['magnitude'], sdss_data['dered_r'], color='r', label='SDSS-r')
+plt.plot(sdss_data['magnitude'], sdss_data['dered_u'], color='b', label='SDSS-u')
+plt.xlim(16, 24)
+plt.yscale('log')
+plt.xlabel(r'$\mathrm{Apparent\,Magnitude}$')
+plt.ylabel(r'$\mathrm{N} \, [\mathrm{deg}^{-2} \, \mathrm{mag}^{-1}]$')
+plt.legend()
+plt.show()
+
+# %%
+# References
+# ----------
+#
+# .. [1] M. R. Blanton and S. Roweis, 2007, AJ, 125, 2348
+# .. [2] J. Herbel, T. Kacprzak, A. Amara, A. Refregier, C.Bruderer and
+#        A. Nicola 2017, JCAP, 1708, 035
+# .. [3] K. N. Abazajian et al. 2009, ApJS, 182, 543
```

Comment:
Note: This looks good here but in the documentation there is no space between `[1]` and `M` and between `[3]` and `K`.